### PR TITLE
Ranged balance tweak

### LINKED ISSUE #22002: Dispersion cap is hidden from the player
`player::get_weapon_dispersion` has this line:
```
int weapon_dispersion = std::max( obj.gun_dispersion() / 3, 45 );
```

This means that any weapon with dispersion below 135 still shoots at 135 dispersion. This is not a bad thing on its own, the problem is that the cap is applied in `player::get_weapon_dispersion` and not `item::gun_dispersion`.

### LINKED ISSUE #22064: Occasional test failures in ranged_test.cpp
Game version: Everything since 41e8209cc106c735436f4a9427f3acf254936428

Operating system: All that run tests.

Tiles or curses: N/A

Mods active: N/A

### Expected behavior
All tests pass consistently.

### Actual behavior
Very occasional (so far 3 / 270 executions in travis) failures in either the "unskilled shooter with an inaccurate pistol" or "unskilled shooter with an inaccurate smg" tests.

### Steps to reproduce the behavior
Run ranged unit tests repeatedly.

### Failures
https://travis-ci.org/CleverRaven/Cataclysm-DDA/jobs/281157304
https://travis-ci.org/CleverRaven/Cataclysm-DDA/jobs/282020818
https://travis-ci.org/CleverRaven/Cataclysm-DDA/jobs/280275187

### PR DESCRIPTION
**Pull request by kevingranade**
Fixes #22064 
Fixes #22002 
A few simple tweaks to prevent unit test failures.

### CODE REVIEW

File: src/character.cpp [hunk 1]
```diff
@@ -1626,13 +1626,13 @@ int Character::get_int_bonus() const
...
     ///\EFFECT_DEX <20 increases ranged penalty
-    return std::max( ( 20.0 - get_dex() ) * 2.25, 0.0 );
+    return std::max( ( 20.0 - get_dex() ) * 2.5, 0.0 );
 }
 
 int Character::ranged_per_mod() const
 {
     ///\EFFECT_PER <20 increases ranged aiming penalty.
-    return std::max( ( 20.0 - get_per() ) * 2.25, 0.0 );
+    return std::max( ( 20.0 - get_per() ) * 2.0, 0.0 );
```
Comment (Coolthulhu) [inline src/character.cpp]: As a balance tweak, it is a regression, as it worsens an existing problem of dexterity vs perception balance.

Comment (kevingranade) [reply src/character.cpp]: This is a non-issue.

Comment (Coolthulhu) [reply src/character.cpp]: You named the PR "Ranged balance tweak", not "Make unit test failure less likely" (at the cost of worsening ranged balance).

Comment (kevingranade) [reply src/character.cpp]: You misunderstand, what I disagree with is that, "dexterity vs perception balance" as you state it is an issue. I humored your opinion on this previously, but *The change you requested broke the tests*, therefore I'm reverting it.

Comment (Coolthulhu) [reply src/character.cpp]: You adjusted the tests heavily before in order to make them fit the data. Now you treat them as the ultimate guide, more important than actual game balance the tests were supposed to be enforcing.
This is obviously wrong. If they can be loosened to fit problems in implementation, they surely can be loosened to avoid regressions in game balance - something they are supposed to be enforcing.

Comment (kevingranade) [reply src/character.cpp]: > You misunderstand, what I disagree with is that, "dexterity vs perception balance" as you state it is an issue.

Comment (Coolthulhu) [reply src/character.cpp]: So you're saying that it's OK that balance of stats is objectively messed up?

Comment (kevingranade) [reply src/character.cpp]: > https://github.com/CleverRaven/Cataclysm-DDA/issues/21998#issuecomment-332700803

> Having stats be roughly similar in utility is a valid goal, but we have no principle that states they must have some particular relationship within a given feature.

Comment (Coolthulhu) [reply src/character.cpp]: Balancing things in a vacuum is a part of the problem, not an explanation for why the problem isn't really a problem.
The tests were supposed to enforce balance with the assumption that the characters "doing the testing" are ones that might be encountered in the game. By adding the utterly pointless dexterity effect to accuracy, you broke that assumption, now the characters are just debug characters. And now you're worsening that discrepancy between debug and real characters - discrepancy that the tests don't account for.

Also, side note, relevant because you are also adjusting a gunmod here:
By giving the weak character a christmas tree gun (inaccessible early on), you made the tests no longer cover actual early game characters. By giving mostly barely modded guns to strong one, you prevented the tests from covering the "endgame".

Basically, at this point the tests mostly test themselves, which at best just roughly correlates with in-game effects. It's safe to adjust the tests themselves instead of worsening the actual gameplay.

Comment (kevingranade) [reply src/character.cpp]: > Balancing things in a vacuum is a part of the problem, not an explanation for why the problem isn't really a problem.

Did I say it isn't a problem? No, but the more pressing *immediate* problem is that the tests are failing, that's broken *right now*.
> The tests were supposed to enforce balance with the assumption that the characters "doing the testing" are ones that might be encountered in the game. 
> By giving the weak character a christmas tree gun (inaccessible early on), you made the tests no longer cover actual early game characters. By giving mostly barely modded guns to strong one, you prevented the tests from covering the "endgame".

As I said in the original pr, https://github.com/CleverRaven/Cataclysm-DDA/pull/21468#issuecomment-327669701
> Re: Assumptions like, "weak gunner is using no mods". I'm trying to balance the system as a whole here, I don't want this blocked on the extrordinarally large set of changes necessary to rebalance all the individual items, so I'm deferring that to a later PR.
This PR provides a very large and coarse adjustment to aiming balance as a whole, baking that balance into individual item stats is going to take a long time and shouldn't block the more coarse adjustment.

To expand on this, various item stats need to be adjusted so that the test NPCs are more representative of real player loadouts at their various levels, I explicity deferred this because it is going to be an extensive slog of adjusting numbers, and is not necessary for 0.D.

In short, it's not done, AFAIK there was never any assertion that it was going to be "done" as of #21468.

Anyway, this is obviously going nowhere, so I'm going to stop rehashing it now.
